app/gui2/index.html

```diff
@@ -19,5 +19,7 @@
     <div id="enso-dashboard" class="enso-dashboard"></div>
     <div id="enso-chat" class="enso-chat"></div>
     <script type="module" src="/src/entrypoint.ts"></script>
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-CLTBJ37MDM"></script>
```

Comment:
Can we make it possible to not include gtm in dev mode?

Reply:
Also, I think it makes sense to grab gtag ID from .env, because we might want to use different ids for different environments

Reply:
re: moving it to .env - makes sense, but maybe after the environment variable is added to the GitHub runners -w which may mean it'll have to be done in another PR.

the dev mode thing can be done in this PR though.

Reply:
on second thought: *for now, as a stopgap*, i think it's better to unconditionally include the tag id. when it gets added to the builders in GH Actions, then we can *omit* them in the `.env` files, and manually provide the environment variable when we want to test Google Analytics. thoughts?

Reply:
Agree